_posts/codelabs/2018-03-04-api-versioning-et-retro-compatibilite-avec-symfony/step1.md

```diff
@@ -0,0 +1,28 @@
+Nous allons donc avoir besoin de spécifier les changements de rétro-compatibilité à appliquer lorsqu'une version précédente est demandée.
+
+Il nous faut alors implémenter une liste des versions dans la configuration de Symfony avec, pour chaque version, le namespace complet du fichier qui contient les versions à appliquer.
+
+# Spécifions les versions rétro-compatibles
+
+Editez alors le fichier `app/config/parameters.yml` de votre projet (ou `config/services.yaml` sous Symfony 4) et ajoutez-y l'entrée suivante sous `parameters` :
+
+```yaml
+parameters:
+    versions:
+        1.1.0: Acme\VersionChanges\VersionChanges110
+        1.0.0: Acme\VersionChanges\VersionChanges100
+        0.9.0: Acme\VersionChanges\VersionChanges009
+        0.8.0: Acme\VersionChanges\VersionChanges008
+```
+
```

Comment:
note: no conf for current v1.2.0

Reply:
Merci, j'ai ajouté la petite note d'information 👍